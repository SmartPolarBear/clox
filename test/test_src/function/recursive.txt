R"(
fun fib(n){
    if(n==1 or n==2){
        return 1;
    }
    return fib(n-1)+fib(n-2);
}

for(var i=1;i<10;i=i+1){
    print fib(i);
}
)"