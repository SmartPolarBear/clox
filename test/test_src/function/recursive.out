R"(1.000000
1.000000
2.000000
3.000000
5.000000
8.000000
13.000000
21.000000
34.000000)"
